### PR TITLE
Ah permissions/analysis view set

### PR DESCRIPTION
**Pull request by haneslinger**

### CODE REVIEW

File: seed/tests/test_analyses_views.py [hunk 1]
```diff
@@ -236,3 +249,262 @@ def test_retrieve_view_with_no_output_file(self):
...
+    def test_analysis_list(self):
+        url = reverse_lazy('api:v3:analyses-list') + "?organization_id=" + str(self.org.id)
+
+        # child user cannot
+        self.login_as_child_member()
+        response = self.client.get(url, content_type='application/json')
+        assert len(response.json()["analyses"]) == 0
+        assert response.status_code == 200
+
+        # root users can create column in root
```
Comment (kflemin) [inline seed/tests/test_analyses_views.py:313]: maybe this is just the wrong comment line?

File: seed/tests/test_analyses_views.py [hunk 2]
```diff
@@ -236,3 +249,262 @@ def test_retrieve_view_with_no_output_file(self):
...
+
+    def test_analysis_get(self):
+        url = reverse_lazy('api:v3:analyses-detail', args=[self.root_analysis.pk]) + "?organization_id=" + str(self.org.id)
+
+        # child user cannot
+        self.login_as_child_member()
+        response = self.client.get(url, content_type='application/json')
+        assert response.status_code == 404
+
+        # root users can create column in root
```
Comment (kflemin) [inline seed/tests/test_analyses_views.py:348]: wrong comment line

File: seed/tests/test_analyses_views.py [hunk 3]
```diff
@@ -236,3 +249,262 @@ def test_retrieve_view_with_no_output_file(self):
...
+
+    def test_analysis_destroy(self):
+        url = reverse_lazy('api:v3:analyses-detail', args=[self.root_analysis.pk]) + "?organization_id=" + str(self.org.id)
+
+        # child user cannot
+        self.login_as_child_member()
+        response = self.client.delete(url, content_type='application/json')
+        assert response.status_code == 404
+
+        # root users can create column in root
```
Comment (kflemin) [inline seed/tests/test_analyses_views.py:361]: same

File: seed/tests/test_analyses_views.py [hunk 4]
```diff
@@ -236,3 +249,262 @@ def test_retrieve_view_with_no_output_file(self):
...
+
+    def test_analysis_start(self):
+        url = reverse_lazy('api:v3:analyses-start', args=[self.root_analysis.pk]) + "?organization_id=" + str(self.org.id)
+
+        # child user cannot
+        self.login_as_child_member()
+        response = self.client.post(url, content_type='application/json')
+        assert response.status_code == 404
+
+        # root users can create column in root
```
Comment (kflemin) [inline seed/tests/test_analyses_views.py:374]: same

File: seed/tests/test_analyses_views.py [hunk 5]
```diff
@@ -236,3 +249,262 @@ def test_retrieve_view_with_no_output_file(self):
...
+        self.login_as_root_member()
+        response = self.client.get(url, content_type='application/json')
+        assert response.status_code == 200
+
+    def test_analysis_stats(self):
+        url = reverse_lazy('api:v3:analyses-stats') + "?organization_id=" + str(self.org.id) + "&cycle_id=" + str(self.cycle.pk)
+
+        # child user cannot
```
Comment (kflemin) [inline seed/tests/test_analyses_views.py:408]: can

File: seed/tests/test_analyses_views.py [hunk 6]
```diff
@@ -236,3 +249,262 @@ def test_retrieve_view_with_no_output_file(self):
...
+        self.login_as_root_member()
+        response = self.client.get(url, content_type='application/json')
+        assert response.status_code == 200
+        assert response.json()["total_records"] == 2
+
+    def test_analysis_get_analyses_for_properties(self):
+        url = reverse_lazy('api:v3:analyses-get-analyses-for-properties') + "?organization_id=" + str(self.org.id)
+        params = json.dumps({"property_ids": [self.root_property.pk, self.child_property.pk]})
+
+        # child user cannot
```
Comment (kflemin) [inline seed/tests/test_analyses_views.py:424]: can

**Review by kflemin — APPROVED** [review]
looks good! can you just review the can/ cannot comment lines in seed/tests/test_analyses_views.py for clarify? thanks!

File: seed/tests/test_analyses_views.py (hunk 6 above)
Comment (haneslinger) [reply seed/tests/test_analyses_views.py:424]: oh! this test shouldn't exist- we are deleting that endpoint.

**Pull request merged**